Modules/Shared/Events/Maid.lua

```diff
@@ -56,19 +56,24 @@ function Maid:__newindex(index, newTask)
 end
 
 --- Same as indexing, but uses an incremented number as a key.
--- @param task An item to clean
--- @treturn number taskId
-function Maid:GiveTask(task)
-	assert(task, "Task cannot be false or nil")
+-- @param tasks Item(s) to clean
+-- @treturn table taskId(s)
+function Maid:GiveTask(...)
+	local taskIds = {}
 
-	local taskId = #self._tasks+1
-	self[taskId] = task
+	for task in pairs({...}) do
```

Comment:
This looks very wrong, this is iterating over the number 1 to the amount of arguments, in any order. 'task' will be a number. The values of the arguments are unused here. Was something like 'for _,task in ipairs{...}' intended?